tests/Rector/Architecture/Factory/NewObjectToFactoryCreateRector/NewObjectToFactoryCreateRectorTest.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Tests\Rector\Architecture\Factory\NewObjectToFactoryCreateRector;
+
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+/**
+ * @covers \Rector\Rector\Architecture\Factory\NewObjectToFactoryCreateRector
+ * @covers \Rector\Rector\Architecture\DependencyInjection\ReplaceVariableByPropertyFetchRector
```

Comment:
This is unneded step. Use just single Rector to do this, so people won't forget

Reply:
Yea, the goal of this annotaiton is to get to tested class quickly. Just control + click in PHPStorm.

In 99 % cases it's in `getRectorClass()` so it's not needed here.